package.json

```diff
@@ -119,6 +119,7 @@
     "lodash": "^4.17.11",
     "marked": "^0.4.0",
     "mermaid": "^8.0.0",
+    "polished": "^3.2.0",
```

Comment:
We're already using [`color`](https://www.npmjs.com/package/color) which has `darken` and `lighten` methods, can we just use that instead of adding another dep?

Reply:
@LucianBuzzo Oh, I missed that.
I went for polished as it was from the Styled component guys

Will use it. 👍